Tests/Framework/TestUtility.cs

```diff
@@ -43,11 +43,11 @@ public virtual void OneTimeTearDown()
             AssetDatabase.Refresh();
         }
 
-        public Scene OpenScene(string path)
+        public Scene OpenScene(string path, OpenSceneMode openMode = OpenSceneMode.Single)
         {
             var tempPath = $"{tempDirectory}/{Path.GetFileName(path)}";
             AssetDatabase.CopyAsset(path, tempPath);
-            return EditorSceneManager.OpenScene(tempPath);
+            return EditorSceneManager.OpenScene(tempPath, openMode);
```

Comment:
Side note for the backport on PB5.2 later -> This method is only available after Unity 2022.3 and later and PB5 is verified for 2019.4 :)

Reply:
Any signature of ` EditorSceneManager.OpenScene` ? Are you sure? https://docs.unity3d.com/2019.4/Documentation/ScriptReference/SceneManagement.EditorSceneManager.OpenScene.html